client/blocks/login/index.jsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import LoginForm from './login-form';
+import TwoFactorAuthentication from './two-factor-authentication';
+import { isTwoFactorEnabled } from 'state/login/selectors';
+
+const Login = ( { twoFactorEnabled, translate } ) => {
+	return twoFactorEnabled
+		? ( <TwoFactorAuthentication /> )
+		: ( <LoginForm
+			title={ translate( 'Sign in to connect to WordPress.com' ) }
```

Comment:
Since this is a block, it should be a more generic string. Something like, "Sign in to WordPress.com" like the linter suggests. We could allow it to be overridden with a prop though.